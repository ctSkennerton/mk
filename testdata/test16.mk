
<./testdata/deps9

test3.mk.o: $deps
	secondprocess $prereq2 $prereq1

one:
	process bar $extracmdarg

two:
	$extracmdarg rebar