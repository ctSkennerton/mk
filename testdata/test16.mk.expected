two: gopher rebar
one: process bar gopher
test3.mk.o: secondprocess two one
